tf/env/local/config-map.tf

```diff
@@ -2,4 +2,14 @@ module "wbaas-config" {
   source = "./../../modules/config-map/wbaas-ui-config"
 
   cname_record = "sites-1.dyna.wbaas.localhost."
-}
\ No newline at end of file
+}
+
+resource "kubernetes_config_map" "storage-bucket" {
+  metadata {
+    name = "storage-bucket"
+  }
+
+  data = {
+    gcs_api_static_bucket_name = "api-assets"
```

Comment:
We probably could jump through a few hoops and deduplicate this (it's repeated in the mino values file). I don't think it'd be worth it though.